### PR TITLE
Hello Microverse! Initial Setup

### PR DESCRIPTION
**Pull request by ebidien**
## Initial Repository
The initial repository was created from the `html&css` linter template

## Initial Setup
The following instructions were followed to complete the initial setup as described in the project requirements instructions:
### Requirements
- Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- Create a new GitHub repo.
- Create a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
- Add a `.gitignore` file.
- Add node_modules to `.gitignore` file.
- Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
- Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- Add 3 files (each one in a separate commit):
  - A `.html` file that includes a header with the text "Hello Microverse!". Use `h1` tag for that.
  - A `.css` file that includes one class with styles for your header (make it in your favorite color).
  - A `test.md` file with the text "This file should be ignored by git".
- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
- Open a pull request.
### Submission
I ensured the above requirements were met to the best of my knowledge before making this pull request

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.heading {
+    text-align: center;
+    color: blue;
+}
```
Comment (binhussen) [inline styles.css:4]: - [REQUIRED ] Please fix your linter error, the `stylelint` is failing. 
to fix this issue you can execute `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x` then execute this `npx stylelint "**/*.{css,scss}" --fix` on your vs code terminal

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @ebidien , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- kindly, fix your linter error, the `stylelint` is failing. be sure all your lint checks are green before submitting your PR.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ebidien** [comment]
@binhussen, I have fixed the error. Please review.

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @ebidien  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️ Professional Pull Request 👍
✔️ No Linter errors 👍  
### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**